app/views/api/v1/targets/_info.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+json.extract! target, :id, :title, :radius, :latitude, :longitude
+
+json.user do
+  json.id target.user.id
+end
+
+json.topic do
+  json.id target.topic.id
+end
```

Comment:
maybe here you can do this:
```
json.extract! target, :id, :title, :radius, :latitude, :longitude, :topic_id, :user_id
```
or if you want to put "user" and "topic", it could be better:
```
json.user target.user_id
json.topic target.topic_id
```